docs/admin/hardware.rst

```diff
@@ -0,0 +1,65 @@
+
+Recommended hardware
+====================
+
+.. include:: ../includes/top-warning.rst
+
+Qubes OS hardware requirements
+------------------------------
+
+In order to install and use SecureDrop Workstation, you will need a Qubes-Compatible computer with the following speifications:
+
+- 64-bit Intel or AMD processor with virtualization support
+- a minimum of 16GB RAM (32GB recommended for production use)
+- sufficient disk space for the Qubes OS base install and SecureDrop Workstation VMs (a 128GB or greater SSD is recommended)
+
+More information on hardware compatibility can be found on the `Qubes OS System Requirements <https://www.qubes-os.org/doc/system-requirements/>`_ page, and information on specific systems can be found via the `hardware compatibility list <https://www.qubes-os.org/hcl/>`_.
+
+In order to print submissions, a supported non-networked printer is required. Supported models currently include:
+
+- TBD
+- Also TBD
+
+More printer options will be added in future releases.
+
+Lenovo ThinkPad T480
+--------------------
+The ThinkPad T480 is a viable option for SecureDrop Workstation, and also has the advantage of being used heavily by the SecureDrop development team. If you plan to use it, you should follow the instructions below to ensure that the BIOS is up to date before proceeding with the installation:
+
+.. _t480_bios:
+
+Upgrading the T480 BIOS
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The instructions below assume the use of a Linux-based computer for the creation of a BIOS upgrade USB. To upgrade the T480 BIOS:
+
+- Locate the model number of the T480 - it can often be found on a sticker on the underside of the laptop, or via the ``Main`` tab in Thinkpad Setup (accessed by pressing **Enter** on startup).
+- Visit `<https://support.lenovo.com>`_ in the Linux-based computer. Type the model number found above into the search bar, then press **Enter**.
+- In the T480 Product Home page, select **Drivers And Software** and choose **BIOS/UEFI**.
+- Expand the **BIOS Update** listing and download the **BIOS Update (Bootable CD)** file.
+- Verify the checksum of the downloaded ISO file using the following command, comparing it against the checksum in the file listing above:
+
+  .. code-block:: sh
+   
+    sha256sum /path/to/downloaded.iso
+
+- Create a USB-bootable version of the ISO using the command:
+
+  .. code-block:: sh
+
+    geteltorito <path/to/CDISO> > usb-bios.iso
+
+- Plug in a USB and check its device name with the ``lsblk`` command - use the root device name below, not a partition (eg. ``/dev/sdc`` instead of ``/dev/sdc1``).
+
+- Write the BIOS update ISO to the USB using the following command:
+
+  .. code-block:: sh
+
+    ``sudo dd if=usb-bios.iso of=/dev/sda bs=1M && sync``
```

Comment:
Sorry, but like in the Qubes docs, I think we'll again have to be a bit more explicit with these instructions, and I would caution against using `/dev/sda` in any example, since it's likely to be their main drive.

Reply:
:heavy_check_mark: